server/src/engineModel/EngineModel.ts

```diff
@@ -58,6 +60,9 @@ export class EngineModel {
             ymlAbstractFunction.detail = `[STATIC] ${ymlAbstractFunction.label}`;
         }
         ymlAbstractFunction.data = `id_${sourceType ? sourceType : 'static'}_${ymlAbstractFunction.label}`;
+        if (!!func.doc) {
+            ymlAbstractFunction.documentation = func.doc[0];
```

Comment:
Why is `doc` an array, again? :thinking: Is it OK to take just `[0]`?

Reply:
Here, `func` is something like an XML node with children. Each child may be present multiple time or just once.
```
<func>
  <myChild/>
  <myChild/>
</func>
```
That's why it is an array. 
We can't have more than one `doc` element in our case under `func`, so it is OK to take just `[0]`.